### PR TITLE
fix: return consistent results when generating uuids

### PR DESCRIPTION
**Pull request by lag-of-death**
## Changes

UUIDs are now never generated in their URN form. This means that doing `coreFormatGenerator('uuid')` will never return a value that starts with `urn:uuid:`. Before, `coreFormatGenerator('uuid')` would sometimes generate a value that starts with `urn:uuid:` and sometimes it doesn't.

## Notes
As in https://github.com/json-schema-faker/json-schema-faker/pull/441, I didn't update the readme nor did I generate the dist files. But maybe I should?

### CODE REVIEW

**Comment by lag-of-death** [comment]
hi there @pateketrueke, could you take a look? I'm not sure if I should generate `dist` files and update the readme. Also, don't know what's wrong with the tests, doesn't seem to be connected to this PR directly (?). And of course, would love to hear your thoughts on the PR itself :+1:

**Comment by pateketrueke** [comment]
Normally it's fine to not include dist-changes until releasing, however there's no problem if your include those in your PR.

Related with tests, I don't understand why they failed yet...

Regarding with the PR I think it's fine, as long it validates and pass our tests.

**Comment by pateketrueke** [comment]
@lag-of-death can you please move the PR to merge against `master` instead?

The `develop` branch is bit unstable and it'll not get merged soon.

**Comment by lag-of-death** [comment]
Thank you @pateketrueke, so now `lag-of-death:fix/generating-uuids` is based off of `master` and the PR is against `master` also. In the end, didn't generate these `dist` files. Please let me know if this needs further adjustments :+1:

**Comment by lag-of-death** [comment]
@pateketrueke, @XVincentX, in the end, the dist files were added

**Comment by XVincentX** [comment]
Nice! @pateketrueke if this is good enough, it'd be great if you could merge and release a new rc; otherwise we'll pull the code from the master branch!

File: src/generators/coreFormat.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const regexps = {
   'json-pointer': `(/(?:${FRAGMENT.replace(']*', '/]*')}|~[01]))+`,
 
   // some types from https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#data-types (?)
-  uuid: '^(?:urn:uuid:)?[0-9a-f]{8}-(?:[0-9a-f]{4}-){3}[0-9a-f]{12}$',
+  uuid: '[0-9a-f]{8}-(?:[0-9a-f]{4}-){3}[0-9a-f]{12}$',
```
Comment (XVincentX) [inline src/generators/coreFormat.js:24]: @pateketrueke It seems like something has broken this. The new file I can find has the same "problem" we fixed some time ago. Is there anything I am missing? I can't find where the specific change happened. https://github.com/json-schema-faker/json-schema-faker/blob/master/src/lib/generators/coreFormat.js#L24

Comment (pateketrueke) [reply src/generators/coreFormat.js:24]: can you check please if the change is present on the develop branch?

Comment (XVincentX) [reply src/generators/coreFormat.js:24]: The develop branch seems to be in line. I really do not understand where this change is coming from

https://github.com/json-schema-faker/json-schema-faker/blob/develop/src/lib/generators/coreFormat.js#L24

Comment (pateketrueke) [reply src/generators/coreFormat.js:24]: Interesting, please open a new issue with the affected schema, thank you!

Comment (XVincentX) [reply src/generators/coreFormat.js:24]: #600 created.

**Pull request merged**